qtawesome dependency bump 1.2.0 breaks builds

First saw this in PR #1095 . Confirmed bumping `qtawesome` to [1.2.0](https://github.com/spyder-ide/qtawesome/releases/tag/v1.2.0) broke tests locally. 

```
=================================== ERRORS ====================================
_________________ ERROR collecting tests/test_translation.py __________________
ImportError while importing test module 'D:\a\invest\invest\tests\test_translation.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_translation.py:12: in <module>
    from natcap.invest import carbon, cli, validation, set_locale, ui_server
C:\Miniconda\envs\test\lib\site-packages\natcap\invest\cli.py:23: in <module>
    from natcap.invest.ui import launcher, inputs
C:\Miniconda\envs\test\lib\site-packages\natcap\invest\ui\inputs.py:19: in <module>
    import qtawesome
C:\Miniconda\envs\test\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py:142: in _import
    return original_import(name, *args, **kwargs)
C:\Miniconda\envs\test\lib\site-packages\qtawesome\__init__.py:28: in <module>
    from .iconic_font import IconicFont, set_global_defaults, FontError
C:\Miniconda\envs\test\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py:142: in _import
    return original_import(name, *args, **kwargs)
C:\Miniconda\envs\test\lib\site-packages\qtawesome\iconic_font.py:27: in <module>
    from qtpy.QtGui import (QColor, QFont, QFontDatabase, QIcon, QIconEngine,
E   ImportError: cannot import name 'QGlyphRun' from 'qtpy.QtGui' (C:\Miniconda\envs\test\lib\site-packages\qtpy\QtGui.py)
```

--- a/.github/workflows/build-and-test.yml
+++ b/.github/workflows/build-and-test.yml
@@ -20,7 +20,7 @@ env:
   # build: dependency of make install
   # nomkl: make sure numpy w/out mkl
   # setuptools_scm: needed for versioning to work
-  CONDA_DEFAULT_DEPENDENCIES: build nomkl setuptools_scm
+  CONDA_DEFAULT_DEPENDENCIES: python-build nomkl setuptools_scm
   LATEST_SUPPORTED_PYTHON_VERSION: "3.10"
 
 jobs:

--- a/requirements-gui.txt
+++ b/requirements-gui.txt
@@ -8,7 +8,7 @@
 # seem better supported by PyInstaller
 
 qtpy>1.3  # pip-only
-qtawesome  # pip-only
+qtawesome<1.2  # pip-only
 requests
 PySide2!=5.15.0  # pip-only
 Flask

